Documentation/README.md

```diff
@@ -10,3 +10,5 @@
 * [Scalebar](Scalebar) - Displays current scale reference.
 * [TemplatePickerViewController](TemplatePicker) - Allows a user to choose a template from a list of `AGSFeatureTemplate` when creating new features.
 * [TimeSlider](TimeSlider) - Allows interactively defining a temporal range (i.e. time extent) and animating time moving forward or backward. Can be used to manipulate the time extent in a MapView or SceneView.
+*[FloorFilter](FloorFilter) - Allows to filter floor plan data in a GeoView by a site, a building in the site, or a floor in the building. 
```

Comment:
The formatting of this is off, if you look at the file in a browser.  The `FloorFilter` line is tacked onto the end of the `TimeSlider` info.  Also, `GeoView` need to be "geo view".  And the list should be in alphabetical order, moved to after `Compass`.